paper-menu-button.html

```diff
@@ -131,7 +131,7 @@
       restore-focus-on-close="[[restoreFocusOnClose]]"
       on-iron-overlay-canceled="__onIronOverlayCanceled">
       <div class="dropdown-content">
-        <content id="content" select=".dropdown-content"></content>
+        <slot id="content"></slot>
```

Comment:
Thanks for pointing that out in `iron-dropdown`, I totally missed that! IMHO, if we're already asking people to change from classes to slot names for `dropdown-trigger`, we should be consistent and ask them to do the same for `dropdown-content`.